### DISCUSSION TITLE
[Question] How to make xref prompt for input when using at blank line

### DISCUSSION THREAD

**Discussion by BooAA** [joaotavora/eglot#736] [Q&A]
I think this is a feature of eglot (I saw the corresponding method in the source code to message "No definition found for...."), but I'm used to move to blank line and press `M-.`, then xref will prompt me for input. How can I reproduce this behavior in eglot?

**Comment by joaotavora** [comment]
Eglot doesn't support this. It might in the future, maybe. Originally, this was a hard limitation of LSP itself, but it seems some servers have surpassed it.

May I also ask if you didn't see the issue-creating template? Converting this to a discussion.

**Comment by BooAA** [comment]
> May I also ask if you didn't see the issue-creating template?

Sorry about this, will use issue template next time.

According to [Emacs Manual](https://www.gnu.org/software/emacs/manual/html_node/emacs/Looking-Up-Identifiers.html)
> M-. (xref-find-definitions) shows the definitions of the identifier at point. With a prefix argument, or if there’s no identifier at point, it prompts for the identifier. (If you want it to always prompt, customize xref-prompt-for-identifier to t.)

It's totally understandable that the "no identifier at point" case cannot be supported because of the limitation of LSP. Is it possible to prompt for input if a prefix argument is given? (e.g. `C-u M-.`) This is useful when you want to jump to some known identifiers in other files but forget which file one is the exact one.
